db/migrate/20201216190051_add_policy_id_foreign_key_on_profiles.rb

```diff
@@ -0,0 +1,5 @@
+class AddPolicyIdForeignKeyOnProfiles < ActiveRecord::Migration[5.2]
+  def change
+    add_foreign_key :profiles, :policies
```

Comment:
We would likely need to test this on stage with real data. There might be dangling records that could break the migration.

Reply:
There shouldn't be, but yes, there's no guarantee. Getting prod snapshots in stage is such a hassle.. but I guess we'll do it again.